ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -255,14 +256,38 @@ RightTabBaseView {
                             stack.currentIndex = 1
                         }
                         onSendRequested: (symbol, tokenType) => {
-                                            root.sendModal.preSelectedHoldingID = symbol
-                                            root.sendModal.preSelectedHoldingType = tokenType
-                                            root.sendModal.preSelectedSendType = tokenType === Constants.TokenType.ERC721 ?
-                                                 Constants.SendType.ERC721Transfer:
-                                                 Constants.SendType.ERC1155Transfer
-                                            root.sendModal.onlyAssets = false
-                                            root.sendModal.open()
-                                        }
+                            const collectible = ModelUtils.getByKey(controller.sourceModel, "symbol", symbol)
+                            if (collectible.communityPrivilegesLevel === Constants.TokenPrivilegesLevel.Owner) {
+                                const ownerAccountItem = ModelUtils.get(collectible.ownership, 0)
+                                Global.openTransferOwnershipPopup(collectible.communityId,
```

Comment:
We need to put attention to such calls in the future and (ideally) refactor them. What should be passed to the transfer ownership request here should be just id. All necessary metadata should be fetched on the side handling that request. This component should not be responsible for carrying all those metadata and therefore be tightly coupled with component handling that action.